src/views/activity/activity_index_view.js

```diff
@@ -14,11 +14,15 @@ export default BaseView.extend({
   },
 
   afterRender () {
+    const startIndex = 1;
+
     this.removeSubviews();
-    let items = this.sliceCollection(this.adapter.results, ActivityResultsCollection, 1, 3);
+    let items = this.sliceCollection(this.adapter.results, ActivityResultsCollection, startIndex, 3);
     this.renderCollection(items, ActivityItemView, '.activity-results');
 
     // if there are results then show otherwise hide
-    this.adapter.results.length ? this.show() : this.hide(); // eslint-disable-line no-unused-expressions
+    this.adapter.results.length > startIndex ? this.show() : this.hide(); // eslint-disable-line no-unused-expressions
+
+    window.app.trigger('adjust-height');
```

Comment:
Horribly minor nit: It might be good to have a different name than `adjust-height`, so that you can grep for the webchannel signal and the app pubsub signal without getting the results mixed up. Feel free to ignore this suggestion, though--it's a small codebase and not all that confusing :-)